docs/openstack/persistent-storage/detach-a-volume.md

```diff
@@ -0,0 +1,76 @@
+# Detach A Volume and Attach it to an instance
+
+## Detach A Volume
+
+### Using Horizon dashboard
+
+Once you're logged in to NERC's Horizon dashboard.
+
+Navigate to Project -> Volumes -> Volumes.
+
+To detach a mounted volume by going back to "Manage Attachments" and choosing
+Detach Volume.
+
+This will popup the following interface to proceed:
+
+![Detach a volume](images/detach-volume-from-an-instance.png)
+
+!!! danger "Unable to Detach Volume"
+    If your bootable volume that is attached to a VM then that volume cannot be
+    detached as this is a root device volume. This bootable volume is created when
+    you launch an instance from an **Image** or an **Instance Snapshot**, and the
+    choice for utilizing persistent storage is configured by selecting the **Yes**
+    option for "Create New Volume". If you explicitly chosen as "No" for this option
+    then there will be no attach volume created for the instance instead an **Ephemeral**
+    disk storage is used.
+    ![Launch Instance Set Create New Volume](images/instance-create-new-volume.png)
+
+### Using the CLI
+
+**Prerequisites**:
+
+To run the OpenStack CLI commands, you need to have:
+
+- OpenStack CLI setup, see [OpenStack Command Line setup](../openstack-cli/openstack-CLI.md#command-line-setup)
+  for more information.
+
+#### Using the openstack client
+
+Then use the openstack command line interface to detach the volume from the VM:
+
+    openstack server remove volume test-vm my-volume
```

Comment:
Worth mentioning that test-vm and my-volume need to be encapsulated if they have spaces.